File: Sorting/Bubble_Sort/bubble_sort.c
#include<stdio.h>
int main()
{
    int n,i=0,j;
    printf("Enter the length of List : ");
    scanf("%d",&n);
    int arr[n];
    printf("Enter the numbers in the list : \n");
    while (i<n)
    {
        scanf("%d",&arr[i]);
        i++;
    }
    bubbleSort(arr,n);
}
void bubbleSort(int list[],int n)
{
    for(int i=0;i<n-1;i++)
    {
        for(int j=0;j<n-i-1;j++)
        {
            if(list[j]>list[j+1])
            {
                int temp=list[j];
                list[j]=list[j+1];
                list[j+1]=temp;
            }
        }
    }
    printf("Sorted Array : ");
    for(int i=0;i<n;i++)
    {
        printf(" %d ",list[i]);
    }
}

File: GeeksForGeeks/Sorting/Bubble_Sort/bubble_sort.py
def bubbleSort(list,n):
	for i in range(0,n-1):
		for j in range(0,n-i-1):
			if list[j]>list[j+1]:
				#swap
				temp=list[j]
				list[j]=list[j+1]
				list[j+1]=temp

	print("Sorted Array : ",end=' ')
	print(list)

def main():
	list=[]
	print("Enter the numbers in the list(Enter x to stop): ")
	while True:
		num=input()
		if num=='x':
			break
		list.append(int(num))
	bubbleSort(list,len(list))

if __name__ == '__main__':
	main()
